ingest/src/main/java/com/microsoft/azure/kusto/ingest/AzureStorageHelper.java

```diff
@@ -23,19 +23,29 @@
 import java.nio.file.Paths;
```

Comment:
I changed it to AzureStorageClient, since it acts as a client to the Azure Storage like sending files to as blobs, sending messages to queues, getting blobs from storage...